draft-ietf-quic-transport.md

```diff
@@ -2432,17 +2432,18 @@ The capacity available on the new path might not be the same as the old path.
 Packets sent on the old path MUST NOT contribute to congestion control or RTT
 estimation for the new path.
 
-On confirming a peer's ownership of its new address, an endpoint MUST
+On confirming a peer's ownership of its new address, an endpoint SHOULD
 immediately reset the congestion controller and round-trip time estimator for
 the new path to initial values (see Appendices A.3 and B.3 in {{QUIC-RECOVERY}})
-unless it has knowledge that a previous send rate or round-trip time estimate is
-valid for the new path.  For instance, an endpoint might infer that a change in
-only the client's port number is indicative of a NAT rebinding, meaning that the
-new path is likely to have similar bandwidth and round-trip time. However, this
-determination will be imperfect.  If the determination is incorrect, the
-congestion controller and the RTT estimator are expected to adapt to the new
-path.  Generally, implementations are advised to be cautious when using previous
-values on a new path.
+unless the only change is the other endpoint's port number.  Because
```

Comment:
```suggestion
unless the only change in the peer's address is its port number.  Because
```

Reply:
This sounds like the address _includes_ the port number, which in my mental model it doesn't?

Reply:
[Section 1.2](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#name-terms-and-definitions):
> Address:
> When used without qualification, the tuple of IP version, IP address, and UDP port number that represents one end of a network path.

Reply:
Right, but @janaiyengar's change makes it sound like the _port_ is _in_ the address. Maybe "unless the only change in the peer's endpoint is its port number"?

Reply:
I don't think "endpoint" would be wrong, but "address" per the definition in this document does include the port number.

Reply:
I was proposing two changes:
- we've used endpoint and peer as the two sides. Because you've used "endpoint" for this end, I'm suggesting using "peer" for the other end.
- we've used address to mean IP and port, so saying "the change in the address" was simply to clarify that the "only" is scoped to the address (and not to other things, like CID). Your current text says "the only change is" without saying the only change to what.